src/manifold/include/manifold.h

```diff
@@ -146,9 +146,17 @@ class Manifold {
                            float radiusHigh = -1.0f, int circularSegments = 0,
                            bool center = false);
   static Manifold Sphere(float radius, int circularSegments = 0);
+  static Manifold CreateFromVertsAndTriangles(
+      std::vector<glm::vec3> verts, std::vector<glm::ivec3> triangles);
   static Manifold Extrude(const CrossSection& crossSection, float height,
                           int nDivisions = 0, float twistDegrees = 0.0f,
                           glm::vec2 scaleTop = glm::vec2(1.0f));
+  static Manifold ExtrudeTransforming(const CrossSection& crossSection,
```

Comment:
In general, I consider it a very high bar to accept a change that enlarges our public API. Significant new functionality that can't easily be implemented externally to the library: good chance. Performance improvement: best done while avoiding new APIs, as this makes it harder for new users to discover the optimal way to use the library.